lib/email/styles.rb

```diff
@@ -238,6 +238,7 @@ def format_html
 
       onebox_styles
       plugin_styles
+      dark_mode_styles if SiteSetting.dark_mode_emails_active
```

Comment:
This needs a test (when setting is on, dark styles are included, and vice versa).